app/views/posts/_message.html.haml

```diff
@@ -0,0 +1,3 @@
+.row.alert.alert-success
+  .col-md-12
+    #{ post.author.login_name } is using Growstuff to discuss #{ post.subject } with a community of food gardeners worldwide. We have advice on growing hundreds of different #{link_to 'Crops', crops_url}, and a community from all around the world. #{link_to 'Sign In', new_member_session_url} or #{link_to 'Sign Up', new_member_registration_url} to respond to this post, or to start using Growstuff to track what you're planting and harvesting.
```

Comment:
* Please use ENV["GROWSTUFF_SITE_NAME"] rather than hardcoding "Growstuff".
* Please line-wrap this to approx 80 columns.
* Don't capitalise "Crops"
* Use the shared/signin_signup partial to show the "Sign in or sign up" message